src/govuk/objects/_button-group.scss

```diff
@@ -0,0 +1,89 @@
+@import "../base";
+
+@include govuk-exports("govuk/objects/form-group") {
```

Comment:
@36degrees Guessing this should be `button-group` instead of `form-group`?

Reply:
Well, that explains why form-groups aren't being styled any more 🤦🏻‍♂️  Thanks!